### PR TITLE
switched the base image to debian jessie

### PR DESCRIPTION
**Pull request by travi**
enables the use of `glibc`, required by `node`

### CODE REVIEW

File: verify.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```
Comment (bpholt) [inline verify.sh:1]: Should we add `set -euo pipefail` here, so the overall script fails if any of the `nvm` or `npm` commands fail, or if the environment is somehow not set up correctly?

**Review by bpholt — APPROVED** [review]

File: verify.sh (hunk 1 above)
Comment (travi) [reply verify.sh:1]: good question. i think my assumption was that jenkins was essentially already behaving that way. is that not true for scripts like this?

Comment (bpholt) [reply verify.sh:1]: I wouldn't necessarily assume it does. I'm not sure how those kind of options are inherited by scripts.

Comment (travi) [reply verify.sh:1]: added

**Pull request merged**